### PR TITLE
Split command interfaces

### PR DESCRIPTION
**Pull request by brunodecarvalho**
Purely cosmetic cleanup.

### CODE REVIEW

**Comment by brunodecarvalho** [comment]
@kevin-cantwell @heewa 

selfie-1

File: redis/redis.go [hunk 1]
```diff
@@ -52,85 +115,46 @@ type Commands interface {
...
-	ZIncBy(key string, score float64, value string) (int, error)
-
-	// HyperLogLog
-
-	PFAdd(key string, values ...string) (int, error)
-	PFCount(key string) (int, error)
-	PFMerge(mergedKey string, keysToMerge ...string) (bool, error)
 }
 
-type NoResultCommands interface {
```
Comment (heewa) [inline redis/redis.go]: What happened to this interface? Is it now called `BatchCommands`? If so, should there be an alias to `NoResultCommands` for backward compatibility?

Comment (brunodecarvalho) [inline redis/redis.go]: Nah, we don't use it externally. Not even sure it should be exported.

**Comment by heewa** [comment]
selfie-1

**Comment by brunodecarvalho** [comment]
Thassabigthumbups! Shipping this.

**Pull request merged**